src/app/clusters/power-topology-server/power-topology-server.h

```diff
@@ -0,0 +1,84 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <lib/core/Optional.h>
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <lib/support/CommonIterator.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace PowerTopology {
+
+using namespace chip::app::Clusters::PowerTopology::Attributes;
```

Comment:
So this will pull all of Attributes into the PowerTopology namespace into any file that includes this header, right?  Is that the intent?  I suspect we don't actually want this `using namespace` in a header....

(Also, I am not sure anything below in this file uses this Attributes namespace.)

Reply:
Removed.